python/lsst/ap/pipe/matchSourceInjected.py

```diff
@@ -202,7 +215,7 @@ def _estimateFakesSNR(self, injectedCat, diffIm):
 
         # Add the forced measurement columns to the input catalog
         for column in forcedSources_table.columns:
-            if "Flux" in column:
+            if "Flux" in column or "flag" in column:
```

Comment:
Can we get these columns from the schema set by the forced source task instead? I am concerned that this might copy too many columns, and might miss some.

Reply:
I don't know exactly how to make this be set by the forced source task instead.
Would this be just by re-defining the schema?

Reply:
I misunderstood: I thought this was renaming columns, but I see it is choosing columns to copy over to the injected catalog. That seems fine!